src/Enum.php

```diff
@@ -226,6 +237,8 @@ public static function assertValidValue($value): void
      *
      * @psalm-pure
      * @psalm-assert T $value
+     * @param mixed $value
+     * @return string
```

Comment:
Same here, why add this?

Reply:
Same here. The @return I guess it's just the PhpStorm complaining about.